lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -363,10 +382,14 @@ else
 			local int = self.metadata.Interior.Sounds.Teleport
 			local pos=net.ReadVector()
 			if LocalPlayer():GetTardisData("exterior")==self then
-				self:EmitSound(ext.mat)
-				self.interior:EmitSound(int.mat or ext.mat)
+				if not self:GetData("demat-fast",false) then
+					self:EmitSound(ext.mat)
+					self.interior:EmitSound(int.mat or ext.mat)
+				end
 			else
```

Comment:
Since the else here only contains an if statement can you just turn this into an elseif?